API/NoDocs/Microsoft.Extensions.DependencyInjection/AutoActivationExtensions.cs

```diff
@@ -0,0 +1,57 @@
+// Assembly 'Microsoft.Extensions.DependencyInjection.AutoActivation'
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class AutoActivationExtensions
```

Comment:
Does marking the entire class mark everything as experimental?

Reply:
Yes it does

Reply:
we should do that in the implementation then (move to class level where we are marking all the members)

Reply:
These annotations should be gone by next week as we are scheduled to review this API.